cmd/enduro/main.go

```diff
@@ -544,6 +545,7 @@ func main() {
 				clockwork.NewRealClock(),
 				cfg.Storage.AIPDeletion,
 				storagesvc,
+				pdf.NewPDFCPU(),
```

Comment:
It was not the easiest to understand that `pdf.NewPDFCPU()` returns a `pdf.FormFiller`, `pdf.NewPDFCPUFormFiller()` would be easier to get.